skale/manager_client.py

```diff
@@ -50,10 +51,10 @@ def __init__(self, endpoint, abi_filepath, wallet=None, provider_timeout=30):
         self.web3.middleware_onion.inject(
             geth_poa_middleware, layer=0)  # todo: may cause issues
         self.__contracts = {}
-        self.nonces = {}
+        self.__abi = get_abi(abi_filepath)
```

Comment:
Is this really required. Abi object is relatively big (~0.5 Mb). And I don't see any reason why we should keep persistently.

Reply:
yeah, agree, actually, it's not required